hello_world_on_chain/README.md

```diff
@@ -119,7 +119,7 @@ contract User {
 
 The goal of the [`User`](contracts/User.sol) contract is to verify that a
 transitive attestation of the `helloWorld` function (defined in
-[`../hello_world_attest_fn_call/main.go`](../hello_world_attest_fn_call/main.go))
+[`../hello_world_attest_fn_call/main.go`](../attest_fn_call/main.go))
```

Comment:
The first part of this link still refers to `../hello_world_attest_fn_call` and should be updated to `../attest_fn_call`